org.eclipse.jdt.ui/ui/org/eclipse/jdt/internal/ui/callhierarchy/FiltersDialog.java

```diff
@@ -118,28 +121,27 @@ private void createTestCodeArea(Composite parent) {
 		layout.numColumns= 1;
 		radioGroup.setLayout(layout);
 
-		fShowAll= new Button(radioGroup, SWT.RADIO);
-		fShowAll.setText(CallHierarchyMessages.FiltersDialog_ShowAllCode);
-
-		fHideTest= new Button(radioGroup, SWT.RADIO);
-		fHideTest.setText(CallHierarchyMessages.FiltersDialog_HideTestCode);
+		for (CallHierarchyFilterOptions op : CallHierarchyFilterOptions.values()) {
+			Button b= new Button(radioGroup, SWT.RADIO);
+			b.setText(op.getText());
+			b.setData(KEY_CALL_HIERARCHY_FILTER_OPTION, op);
+			buttons.add(b);
+		}
 
-		fShowTest= new Button(radioGroup, SWT.RADIO);
-		fShowTest.setText(CallHierarchyMessages.FiltersDialog_TestCodeOnly);
 		setSelection();
 
 		GridData gridData= new GridData();
 		gridData.horizontalIndent= 0;
-		fShowAll.setLayoutData(gridData);
-		fHideTest.setLayoutData(gridData);
-		fShowTest.setLayoutData(gridData);
+
+		for (Button button : buttons) {
+			button.setLayoutData(gridData);
+		}
 	}
```

Comment:
I would rewrite this function to this:

```suggestion
    private void createTestCodeArea(Composite parent) {
		Composite radioGroup= new Composite(parent, SWT.NONE);
		radioGroup.setLayoutData(new GridData(GridData.FILL_HORIZONTAL));
		GridLayout layout= new GridLayout();
		layout.numColumns= 1;
		radioGroup.setLayout(layout);

		GridData gridData= new GridData(GridData.FILL_HORIZONTAL);
		gridData.horizontalIndent= 0;

		for (CallHierarchyFilterOptions op : CallHierarchyFilterOptions.values()) {
			Button b= new Button(radioGroup, SWT.RADIO);
			b.setText(op.getText());
			b.setData(KEY_CALL_HIERARCHY_FILTER_OPTION, op);
			b.setLayoutData(gridData);
			buttons.add(b);
		}
		setSelection();
	}
```

That would fix the trimming of the labels too.

Reply:
This is not a refactoring, it adds/changes the following lines:
- `radioGroup.setLayoutData(new GridData(GridData.FILL_HORIZONTAL));`
- `GridData gridData= new GridData(GridData.FILL_HORIZONTAL);`
- `b.setLayoutData(gridData);`

This should be done in another PR.

Reply:
> This should be done in another PR.

Would that be OK @gzsombor ?